app/src/main/java/ch/epfl/sdp/blindly/user/User.kt

```diff
@@ -9,24 +9,25 @@ import kotlinx.serialization.Serializable
 import java.time.LocalDate
 import java.time.Period
 
+const val SIZE_2 = 2
```

Comment:
I'm not really sure of this naming, what is the point of having the constant :D Maybe something like SIZE_LAT_AND_LNG ? I leave it up to you.